### ISSUE TITLE
record count inconsistent with docs

### ISSUE THREAD

**Issue by floswald** [floswald/psidR#2]
Florian,

I'm using your psidR package for my honors senior thesis here at the University of Michigan, and I was wondering if you could help me out with the discrepancies between the PSID documented record counts and the record counts outputted in the verbose output of build.panel.

For example, in the 2007 dataset, when heads.only is true and core is false, I get 8,597 observations. But in the documentation (attached) there should only be 8,289 records. The analogous numbers for 2009 are 9,005 and 8,690.

Do you have any suggestions on why this might be? Thank you very much for your time!

Best Regards,

Yichuan Wang

**Comment by floswald** [comment]
Florian,

I think I figured out the head count issue. I think it's because you're not looking at the sequence number in addition to "relation to head". From the individual data documentation for "2009 Relation to Head" (ER34003)

"Note that these relationships are those to the 2007 Head for any individual whose 2009 sequence number (ER34002) is greater than 50, that is, has moved out of the FU. Thus, for example, if the 2007 Head is no longer present at the time of the 2009 interview, his or her relationship to Head is coded 10; the new 2009 Head also is coded 10. Therefore, to select current Heads, the user must select those coded 10 in this variable whose sequence numbers (ER34002) are coded 01."

Therefore when you filter for the relationship to heads, you still leave the heads who are no longer heads but still coded as such. The way you have it still might be useful because it allows people to track families over time (if the father leaves the family, you still might want to count it as the same family).

I also found another bug when I tried to request three individual variables, and there was some "name setting" process that broke the program.

I will try to post an issue when I have some time, but I thought you might be interested in these issues.

Best Regards,

Yichuan Wang
